payment/uc_credit/tests/uc_credit.tests.info

```diff
@@ -1,5 +1,5 @@
 [UbercartCreditCardTestCase]
-name = Credit cards with Test Gateway
+name = Credit cards With Test Gateway
```

Comment:
Should it be Credit Cards With Test Gateway?
(Capital C in Cards)

Reply:
Actually, it should be neither! Although module names get All Words Capitalized, the prevailing practice in core for test names seems to be to only capitalize the first letter ("Credit cards..."), as well as any proper names ("Test Gateway" is the name of a module). I was not consistent about that, so I've updated the PR.